.vscode/launch.json

```diff
@@ -38,15 +38,23 @@
         "--hg-key=$HGAPIKEY"
       ]
     },
+    {
+      "name": "Run Server (query GRPC Storage Server)",
```

Comment:
@gamab this is cool.

Can I ask why this has the word `query` in it? I'm in Datasources Core Services, where we own the DS Querier component - the multi-tenant Datasource Querier. Long shot, but I'm wondering whether the word `query` here was simply copy-pasted from the example directly above `Run API Server (query-localhost)`, but could in fact be removed from here?